rwengine/src/render/OpenGLRenderer.hpp

```diff
@@ -75,6 +75,8 @@ class Renderer {
         Textures textures;
         /// Alpha blending state
         bool blend;
+        /// Blending mode 0 - glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA); 1 - glBlendFunc(GL_ONE, GL_ONE);
+        size_t blendMode;
```

Comment:
Please create a new enum if we are going to support multiple blending modes. We could even consider removing the bool for blending and using the enum values to disable blending.

Reply:
Good idea. Thank you